src/DynamoCore/Search/Legacy/CategoryBuilder.cs

```diff
@@ -0,0 +1,420 @@
+﻿using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using System.Linq;
+using Dynamo.Search.SearchElements;
+using Dynamo.UI;
+
+namespace Dynamo.Search
+{
+#if false
```

Comment:
This and `BrowserInternalElementForClasses` are Sitrus only legacy files. I will remove it if you need.